assets/scss/_code.scss

```
@@ -1,101 +1,84 @@
 // Code formatting.
 
 .td-content {
-	// Highlighted code.
-    .highlight {
-        @extend .card;
-	
-        margin: 2rem 0;
-        padding: 0;
-        position: relative;
+  // Highlighted code.
+  .highlight {
+    @extend .card;
 
-        .click-to-copy {
-            display: block;
-            text-align: right;
-            height: 1ex;
-        }
-
-        pre {
-            margin: 0;
-            padding: 1rem;
+    margin: 2rem 0;
+    padding: 0;
+    position: relative;
 
-            // Default click-to-copy button
-
-            button {
-               position: absolute;
-               color: $gray-400;
-               border-radius: 3px;
-               border-width: 0;
-               background-color: inherit;
-               box-shadow: 1px 1px $gray-400;
-               right: 8px;
-               top: 6px;
+    .click-to-copy {
+      display: block;
+      text-align: right;
+      height: 1ex;
+    }
 
-               &:hover {
-                  color: $dark;
-                  background-color: $gray-400;
-               }
-               &:active {
-                  color: $dark;
-                  background-color: $gray-400;
-                  transform: translateY(2px);
-               }
-            }
+    pre {
+      margin: 0;
+      padding: 1rem;
 
-            .c2c-tooltip {
-               background: $dark;
-               color: $white;
-               padding: 2px 4px;
-               border-radius: 3px;
-               display: block;
-               visibility: hidden;
-               opacity: 0;
-               transition: visibility 0s, opacity 0.5s linear;
-            }
+      // click-to-copy button
+      button {
```

I'd prefer that you use a class name specific to this feature (and add the class via the js):

```suggestion
      button.td-click-to-copy {
```

---

Updated her and added the class in the js file